crdt.go

```diff
@@ -425,7 +425,10 @@ func (store *Datastore) rebroadcastHeads() {
 	store.seenHeadsMux.RUnlock()
 
 	// Send them out
-	store.broadcast(headsToBroadcast)
+	err = store.broadcast(headsToBroadcast)
+	if err != nil {
+		store.logger.Warn("broadcast failed: %v", err)
+	}
```

Comment:
I'm moderately sure pubsub will already complain if this happens, so we're just reprinting errors.

Reply:
But aren't we trying not to assume a broadcaster implementation in this project?

Reply:
Fair enough, can take this then.